### ISSUE TITLE
Does SketchupFFI#initialize override Object#initialize ?

### ISSUE THREAD

**Issue by DanRathbun** [jimfoltz/jf-sketchup-ffi#1]
Does `SketchupFFI#initialize` override `Object#initialize` when included into the main `ObjectSpace` ?

Ie, in this example file ...
https://github.com/jimfoltz/jf-sketchup-ffi/blob/master/examples/attr-dict-read.rb
.... the code includes the `SketchupFFI` module which has both a module function `::initialize` and an instance method `#initialize`. Including a mixin module with instance methods causes these instance methods to be added to the mixee as private instance methods.

Perhaps it is not best practice to give example's including a library module into the main `ObjectSpace` ? How about wrapping the example's in a namespace example module ?

**Comment by jimfoltz** [comment]
Yes it does, and I had changed that locally to `SketchupFFI.initialize` but didn't push it to github.

I agree about namespacing - will get that taken of.

Thanks.
